### ISSUE TITLE
Windows Installer and back-end tests are wrong!

### ISSUE THREAD

**Issue by HMarzban** [ether/etherpad-lite#5200]
Changing directory on [`installOnWindows.bat`](https://github.com/ether/etherpad-lite/blob/develop/src/bin/installOnWindows.bat) file in lines [4](https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/src/bin/installOnWindows.bat#L4) and [19](https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/src/bin/installOnWindows.bat#L19) are wrong!

**[Reason/Solution]:** we need to go back one more directory to reach etherpad-lite root.

This line:

https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/src/bin/installOnWindows.bat#L3-L4

Must change to:

```yml
cd /D "%~dp0..\.."
```

**Also,**

Github action flow on [`backend-tests.yml`](https://github.com/ether/etherpad-lite/blob/develop/.github/workflows/backend-tests.yml) in lines [119](https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/.github/workflows/backend-tests.yml#L119) and [175](https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/.github/workflows/backend-tests.yml#L175) are wrong.

**[Reason/Solution]:** 

These lines:

https://github.com/ether/etherpad-lite/blob/4d2839457a903d210e9790e43e0b353fa9e1eb6c/.github/workflows/backend-tests.yml#L118-L121

Must change to:
```yml
      - name: Install all dependencies and symlink for ep_etherpad-lite
        run: src/bin/installOnWindows.bat
```

**Comment by JohnMcLear** [comment]
So hold on, the backend tests on windows were broken but never failed?

**Comment by HMarzban** [comment]
That's right, I saw Etherpad action! But this yml configuration does not run on our github action correctly!

![image](https://user-images.githubusercontent.com/20157508/135067664-c17f2268-00b6-4d42-8ad2-f052168d89b5.png)

But in general `installOnWindows.bat` is broken because the `bin` folder has been changed and moved to the `src` directory.

**Comment by JohnMcLear** [comment]
How very strange...  Do you want to send a PR w/ your proposed fixes and then we can ensure it runs properly within test environments?

**Comment by HMarzban** [comment]
Sure

**Comment by rhansen** [comment]
Fixed by #5205.
